components/resources/brave_components_resources.grd

```diff
@@ -99,8 +99,8 @@
       <if expr="not is_android">
         <part file="brave_shields_resources.grdp" />
         <part file="brave_private_new_tab_resources.grdp" />
-        <part file="../ai_chat/resources/page/ai_chat_ui_resources.grdp" />
       </if>
+      <part file="../ai_chat/resources/page/ai_chat_ui_resources.grdp" />
```

Comment:
I'm going to check what particular string do we need on Android when handlers and the whole WebUI is visible.

Reply:
all of them I'd guess